src/constants.js

```diff
@@ -2,8 +2,25 @@ exports.DSTU2 = {
 	RESOURCE_TYPES: {
 		OPERATIONOUTCOME: 'OperationOutcome',
 		CONFORMANCE: 'Conformance',
+		ALLERGYINTOLERANCE: 'AllergyIntolerance',
```

Comment:
Yea we talked about this on the last PR and this is still a temporary necessary evil. Currently resources do not have a static type property, so to pull the resource type from a resource we need to create an instance (this means creating an instance of every resource every time metadata is requested). We could attempt to parse this from the filename but these are not using globs and being loaded all at once and this may assume the type does not change across all versions. Each individual profile has a conformance file used to generate conformance statements for the metadata request. That is where we need to load a specific resource for a specific type and pull out it's type property. I talked about this briefly yesterday, but we are going to update the generate resource script to add a static getter for types to make this easy. Once we have that setup, the resource type constants are no longer necessary.